scripts/migration/js2ts

```diff
@@ -80,72 +80,82 @@ for (const filename of filenames) {
   contents = contents.replace(/^\s*["']use strict["']\s*; *\n/m, '');
 
   // Migrate from goog.module to goog.declareModuleId.
-  const closurePathRelative =
-      path.relative(path.dirname(path.resolve(filename)), closurePath);
+  const closurePathRelative = path.relative(
+    path.dirname(path.resolve(filename)),
+    closurePath,
+  );
   contents = contents.replace(
-      /^goog.module\('([$\w.]+)'\);$/m,
-      `import * as goog from '${closurePathRelative}/goog.js';\n` +
-          `goog.declareModuleId('$1');`);
+    /^goog.module\('([$\w.]+)'\);$/m,
+    `import * as goog from '${closurePathRelative}/goog.js';\n` +
+      `goog.declareModuleId('$1');`,
+  );
 
   // Migrate from goog.require to import.
   contents = contents.replace(
-      requireRE,
-      function(
-          orig,      // Whole statement to be replaced.
-          name,      // Name of named import of whole module (if applicable).
-          names,     // {}-enclosed list of destructured imports.
-          type,      // If truthy, it is a requireType not require.
-          moduleId,  // goog.module ID that was goog.require()d.
-      ) {
-        const importPath = modulePaths[moduleId];
-        type = type ? ' type' : '';
-        if (!importPath) {
-          console.warn(`Unable to migrate goog.require('${
-              moduleId}') as no ES module path known.`);
-          return orig;
-        }
-        let relativePath =
-            path.relative(path.dirname(path.resolve(filename)), importPath);
-        if (relativePath[0] !== '.') relativePath = './' + relativePath;
-        if (name) {
-          return `import${type} * as ${name} from '${relativePath}';`;
-        } else if (names) {
-          names = names.replace(keyValueRE, '$1 as $2');
-          return `import${type} ${names} from '${relativePath}';`;
-        } else {  // Side-effect only require.
-          return `import${type} '${relativePath}';`;
-        }
-      });
+    requireRE,
+    function (
+      orig, // Whole statement to be replaced.
+      name, // Name of named import of whole module (if applicable).
+      names, // {}-enclosed list of destructured imports.
+      type, // If truthy, it is a requireType not require.
+      moduleId, // goog.module ID that was goog.require()d.
+    ) {
+      const importPath = modulePaths[moduleId];
+      type = type ? ' type' : '';
+      if (!importPath) {
+        console.warn(
+          `Unable to migrate goog.require('${moduleId}') as no ES module path known.`,
```

Comment:
Darn it, Prettier, you don't have to accept my approach to exceeding line length, but could you at least provide a better alternative?!

Reply:
Could you actually fix this, please?

Also, do you know why this file has been reformatted in this PR, and not earlier?

Reply:
I think since we have a system for autoformatting, it makes more sense to let it do the formatting, even if we don't 100% agree with it. This saves everybody time.

> Also, do you know why this file has been reformatted in this PR, and not earlier?

Not sure, maybe because it doesn't have a file extension it wasn't being detected previously? We did update to prettier 3.1 recently, but I can't find anything relevant in the [release notes](https://prettier.io/blog/2023/11/13/3.1.0.html#other-changes).